example/cluster-stacks/openstack/ferrol/node-images/registry.yaml

```diff
@@ -4,5 +4,6 @@ config:
   bucket: <bucket_name>
   accessKey: <access_key>
   secretKey: <secret_key>
+  # projectID: <openstack_project_id> # Needs to be specified when type is equal to Swift
```

Comment:
If I read the code correctly, `projectID` is necessary only when type equals Swift and URL is not set in `config.yaml`.

Reply:
Yes, that is right.